tasks/configure.yml

```diff
@@ -17,12 +19,14 @@
   template:
     src: "ood_portal.yml.j2"
     dest: "{{ ood_base_conf_dir }}/ood_portal.yml"
+    mode: 'u=rw,g=r,o=r'
```

Comment:
Can we change this to 640? It's got the same info the file above has.

Reply:
Fixed in latest commit.  I wasn't sure if the puns need to look at this file.  For some reason this ended up as 644 in my test deployment previously.